packages/coverage-c8/src/index.ts

```diff
@@ -1,6 +1,11 @@
-export * from './takeCoverage'
+import * as coverage from './takeCoverage'
 
-export async function getProvider() {
-  const { C8CoverageProvider } = await import('./provider')
-  return new C8CoverageProvider()
+export default {
+  ...coverage,
+  async getProvider() {
```

Comment:
We should be throwing an error if this is called in `--browser` mode. Is there some global flag that could be used easily?

```ts
if (globalThis.__vitest_browser_or_something__)
  throw new Error('@vitest/c8 does not work with --browser (yet). Use @vitest/istanbul instead.')
```

Reply:
No, there is no flag currently. But I would probably throw error sooner on the node side with `ctx.isBrowserEnabled` check.

Reply:
The config checking will now throw error: 752fcd2